repentogon/resources/scripts/main_ex.lua

```diff
@@ -447,7 +447,7 @@ local typecheckWarnFunctions = {
 		["table"] = checkTableSizeFunction(4)
 	},
 	[ModCallbacks.MC_POST_PICKUP_SELECTION] = {
-		["table"] = checkTableSizeFunction(2)
+		["table"] = checkTableTypeFunction({ "integer", "integer", "boolean"})
```

Comment:
Does this trigger a warning if you don't provide the boolean?

Reply:
No, it doesn't, tough now that I've checked it seems that neither this, nor the previous function, actually trigger a warning if you pass less than the specified arguments, only if you pass more (and in the current case if you pass an argument that does not of the specified type).